pyccel/ast/numpyext.py

```diff
@@ -805,6 +808,10 @@ def _set_order(self, x1, x2):
         else:
             self._order = 'C'
 
+    @property
+    def is_elemental(self):
+        return True
+
```

Comment:
Do we need this property here? The parent class `NumpyUfuncBase` already has it.

Reply:
Ah no! I confused `NumpyUfuncUnary` and `NumpyUfuncBase`

Reply:
Fixed in aa02107